vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -1392,7 +1442,19 @@ def _read_blocks(
         if num_local_blocks == 0:
             remote_rank = self.tp_rank // tp_ratio
             agent_name = self._remote_agents[dst_engine_id][remote_rank]
-            self.nixl_wrapper.send_notif(agent_name, notif_msg=notif_id)
+            try:
+                self.nixl_wrapper.send_notif(agent_name, notif_msg=notif_id)
+            except nixlExceptions as e:
+                logger.warning(
```

Comment:
and then no need to include `e` in the message
```suggestion
                logger.exception(
```